templates/diff/templates/main.html

```diff
@@ -0,0 +1,75 @@
+{% extends "layout.html" %}
+{% import "macros.html" as mc %}
+
```

Comment:
this soon become unmanageable, we need to upgrade this to an automatically generated report template.
Your patience is well appreciated :)

Reply:
I propose a config for the config! that will config what the config will put into the `main.html` content

Reply:
The config Mihai recommends is called Application Profile definition. This will follow in the Semantic Web learning sessions.